testSuite/shared/src/test/scala/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -0,0 +1,12 @@
+package testsuite.javalib.lang
+
+import locales.Implicits._
+
+class CharacterTest extends munit.FunSuite {
+  val hellowWorldLower: String = "helloworld"
+  val helloWorldUpper = "HELLOWORLD"
+
+  // Used to swallow unused imports warning on JVM platform
+  void(hellowWorldLower)
```

Comment:
@cquiroz this is the best idea I had for unused imports, still requires intervention which sucks.